### PR TITLE
CVE-2021-35515, CVE-2021-35516, CVE-2021-35517, CVE-2021-36090: upgrading commons-compress

### PR DESCRIPTION
**Pull request by davidrabinowitz**

### CODE REVIEW

File: spark-bigquery-parent/pom.xml [hunk 1]
```diff
@@ -244,6 +244,11 @@
                 <artifactId>arrow-vector</artifactId>
                 <version>${arrow.version}</version>
             </dependency>
+            <dependency>
```
Comment (VinayLondhe14) [inline spark-bigquery-parent/pom.xml:247]: Is this a transitive dependency that we are fixing to a particular version?

Comment (davidrabinowitz) [reply spark-bigquery-parent/pom.xml:247]: transitive from arrow, even in arrow 7 they haven't upgraded commons-compress, and we are getting security alerts on it

Comment (VinayLondhe14) [reply spark-bigquery-parent/pom.xml:247]: Gotcha

**Review by VinayLondhe14 — APPROVED** [review]

**Pull request merged**